### ISSUE TITLE
Error with non default tics_per_ms

### ISSUE THREAD

**Issue by frankmichler** [nest/nest-simulator#141]
# Error with non default tics_per_ms
## Error description
When compiled with --with-tics-per-ms=4, on NEST import a NEST error
occurs in constructor of node class "correlospinmatrix_detector".

```
Nov 01 19:48:14 SLIInterpreter [Error]: 
    An error occured while loading module NEST Standard Models Module

Nov 01 19:48:14 SLIInterpreter [Error]: 
    InvalidDefaultResolution

Nov 01 19:48:14 SLIInterpreter [Error]: 
    The default resolution of 0.25 ms (= 1 tics = 1 step) is not consistent with the value 0 ms (= 0 tics = 0 steps) of 
    property 'delta_tau' in model UnknownNode.
    This is an internal NEST error, please report it at https://github.com/nest/nest-simulator/issues
```
## Root cause
in correlospinmatrix_detector.cpp, default constructor uses 0.1 for delta_tau, obviously assuming tics-per-ms=10.

```cpp
 nest::correlospinmatrix_detector::Parameters_::Parameters_()
  : delta_tau_( Time::ms( 0.1 ) )
  , tau_max_( 10 * delta_tau_ )
  , Tstart_( Time::ms( 0.0 ) )
  , Tstop_( Time::pos_inf() )
  , N_channels_( 1 )
{
}
```

## Proposed solution
Use 'Time::min()' instead of '0.1'

**Comment by frankmichler** [comment]
Time::min() was wrong. I misunderstood it to be the length of the minimal time step, but it seems to be the lowest possible time value which is a huge negative number.

This should be the solution which gives the lenght of a step in ms.
```cpp
delta_tau_( Time::ms( Time::get_tics_per_step() / Time::get_tics_per_ms() ) )
```
For the default values tics_per_step==100 and tics_per_ms==1000. this should result in 0.1
Implemented in commit 4dcad7abf4b7aac4b944ac426fa43c1064a2113b pull request #144

**Comment by frankmichler** [comment]
Pull request #144 with the proposed solution was merged in commit d971005728061322214dd706adebc8e0d1653991
Therefore I close the issue.
